pandas/tests/io/test_gcs.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from pandas import DataFrame, read_csv
+from pandas.compat import BytesIO
+from pandas.io.common import is_gcs_url
+from pandas.util import _test_decorators as td
+
+
+def test_is_gcs_url():
+    assert is_gcs_url("gcs://pandas/somethingelse.com")
+    assert is_gcs_url("gs://pandas/somethingelse.com")
+    assert not is_gcs_url("s3://pandas/somethingelse.com")
+
+
+@td.skip_if_no('gcsfs')
+def test_read_csv_gcs():
+    try:
```

Comment:
hmm we do install mock in py27, can you move this to pandas.util.testing (and then just import patch from there)

Reply:
I don't think we should make changes to `util/testing.py`. We don't want to require `mock` (for py27), and testing will be awkward if it's in `pandas/util`.

Better to make a little fixture or something for it.

Reply:
@TomAugspurger do you mean a fixture that does use mock, but in another directory that won't get imported outside of testing? Or a fixture that doesn't use mock at all?